src/algebra/hom/group.lean

```diff
@@ -662,30 +667,26 @@ add_decl_doc add_monoid_hom.map_add
 
 namespace monoid_hom
 variables {mM : mul_one_class M} {mN : mul_one_class N} {mP : mul_one_class P}
-variables [group G] [comm_group H]
+variables [group G] [comm_group H] [monoid_hom_class F M N]
 
 include mM mN
 
-@[to_additive]
-lemma map_mul_eq_one (f : M →* N) {a b : M} (h : a * b = 1) : f a * f b = 1 :=
```

Comment:
FYI: this already had a generalized version, and this namespaced one wasn't used anywhere besides the two lemmas below, so I removed it.